src/components/widgetCRUD/services/widgetConfService.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 Cognite AS
```

Comment:
More comments explaining the purpose of this code please.

Reply:
there will be some changes in the PR #86. I'll add the comments there because some conflicts may occur if I changed this now.